### PR TITLE
Clientservice default config

### PR DESCRIPTION
**Pull request by robem**
This PR includes two quality of life changes to improve usage. Please see commit messages for details.

### CODE REVIEW

**Review by shgandhi — APPROVED** [review]
Thanks for the change @robem, and detailed commit messages. LGTM!

File: client/clientservice/src/configuration.cpp [hunk 1]
```diff
@@ -13,28 +13,105 @@
...
+  for (int id = 0; id < 4; ++id) {
+    concord::client::concordclient::ReplicaInfo ri;
+    ri.id.val = id;
+    ri.host = "localhost";
+    ri.bft_port = 3501;
+    ri.event_port = 50051;
+    config.topology.replicas.push_back(ri);
+  }
+
+  config.topology.client_retry_config.initial_retry_timeout = std::chrono::milliseconds(1000);
```
Comment (Liorgal28) [inline client/clientservice/src/configuration.cpp:41]: Not so urgent but this should be:
initial - 3000
min - 3000
max - 5000

**Review by Liorgal28 — APPROVED** [review]
LGTM! just minor things

**Comment by robem** [comment]
@shgandhi @Liorgal28 Thanks for your review! I will implement your suggestion in a follow-up change.

**Pull request merged**